docs/release-notes/8.2.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[release-notes-header-8.2.0]]
+== 8.2
+
+[discrete]
+[[release-notes-8.2.0]]
+=== 8.2.0
+
+[discrete]
+[[deprecations-8.2.0]]
+==== Deprecations
+* Support for the following endpoints will be removed ({pull}129448[#129448]):
+** `detection_engine/rules/_bulk_create`
+** `detection_engine/rules/_bulk_delete`
+** `detection_engine/rules/_bulk_update`
```

Comment:
I think there are a few issues with this section that could be fixed:

- The endpoint URLs are missing the `/api` prefix.
- For each of the endpoints, we could link to the corresponding page in the docs.
- We could set some expectations for when these endpoints could be removed from Kibana in the future.
- We could mention what endpoints our users can use instead of the deprecated ones.

I'd propose maybe something like that:

--------

- The following API endpoints are deprecated ([#129448](https://github.com/elastic/kibana/pull/129448)) and will be removed in the future:
  - [`/api/detection_engine/rules/_bulk_create`](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-create)
  - [`/api/detection_engine/rules/_bulk_delete`](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-delete)
  - [`/api/detection_engine/rules/_bulk_update`](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-update)
  - The deprecated endpoints will be removed from Kibana in one of the next major releases, but not earlier than 18 months.
  - Instead of the deprecated ones, we recommend you to start using the [bulk actions](https://www.elastic.co/guide/en/security/8.1/bulk-actions-rules-api.html#bulk-actions-rules-api-action) endpoint for actions like bulk enabling/disabling, bulk deleting, bulk duplicating or bulk editing rules. You can also use the regular [create](https://www.elastic.co/guide/en/security/8.1/rules-api-create.html), [update](https://www.elastic.co/guide/en/security/8.1/rules-api-update.html) and [delete](https://www.elastic.co/guide/en/security/8.1/rules-api-delete.html) endpoints.